examples/platform/linux/Options.cpp

```diff
@@ -189,6 +195,11 @@ const char * sDeviceOptionHelp =
     "  --wifi-supports-5g\n"
     "       Indicate that local Wi-Fi hardware should report 5GHz support.\n"
 #endif // CHIP_DEVICE_CONFIG_ENABLE_WIFI
+#if (CHIP_DEVICE_CONFIG_ENABLE_WIFI && CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF)
```

Comment:
The other ifdefs all seem to be `#if CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF`.

The conditions should be consistent between the enum definition, the help text, and the code handling the option.

Reply:
I will fix it